test/canvas-mock.js

```diff
@@ -87,4 +90,54 @@ function mockify(window) {
   };
 }
 
+
+/**
+ * Initialize the mock, jsdom and jsdom_global for unit test usage.
+ *
+ * In particular, this takes care of a 'helpful' message about support of `getContext()`, which in
+ * practice is a complete eyesore. It's not a problem in our case, because right here we're using
+ * our own canvas mock to deal with it.
+ * The unit test run fine with or without this message, it just gets in the way.
```

Comment:
This comment could be as short as "Supresses getContext warning, ..."
And in the "..." Post the original message

Reply:
the original message is an exception trace at least 15 lines long, would prefer not to add that in full. The first line of that is in the code.

Reply:
Ok I still think this comment could be shortened considerably

Reply:
OK will shorten.